The-DevOps-Dojo/cv-challenge01
# Feature/dockerize application - Add Dockerization to Application

This PR adds Docker support to the application, enabling containerized deployment for easier scalability and portability.

Key Changes:
- Added Dockerfile for FastAPI application.
- Created docker-compose.yml for multi-service orchestration.
- Updated README with Docker setup instructions.

Testing Steps:
1. Run `docker-compose up` to launch the application in a containerized environment.
2. Verify that the application is accessible on localhost:8000.
